modules/manage/partials/tiered-storage.adoc

```diff
@@ -843,6 +843,8 @@ For additional properties, see <<Tiered Storage configuration properties>>.
 * <<Enable Tiered Storage for a cluster>>
 * <<Enable Tiered Storage for specific topics>>
 
+When you enable Tiered Storage on a topic already containing data, Redpanda uploads any existing data in that topic on local storage to the object store bucket. It will start from the earliest offset available on local disk. It is strongly recommended to avoid repeatedly toggling remote write on and off, because this can result in inconsistent data and data gaps in Tiered Storage for a topic.
```

Comment:
```suggestion
When you enable Tiered Storage on a topic already containing data, Redpanda uploads any existing data in that topic on local storage to the object store bucket. It will start from the earliest offset available on local disk. Redpanda strongly recommended that you avoid repeatedly toggling remote write on and off, because this can result in inconsistent data and data gaps in Tiered Storage for a topic.
```